### PR TITLE
[#6064] Mobile second nav

### LINKED ISSUE #12: Language menu in mobile version
**URL:** https://stage.liqd.net/ and https://liqd.net/
**user:** any
**expected behaviour:** Menu/tab for switching between German and English is clear and operable
**behaviour:** Menu items overlap and sometimes not clickable
**important screensize:** <768 pixel width
**device & browser:** mobile

**Note:** This is not super time-critical and doesn't neccessarily need to be fixed before the django update launch. Can be done in summer.

Screenshot from 2022-04-28 14-59-20
Screenshot from 2022-04-28 15-12-33

### PR DESCRIPTION
**Pull request by phillimorland**
also fixes #12

### CODE REVIEW

File: apps/core/templates/navbar-simple.html [hunk 1]
```diff
@@ -20,38 +20,36 @@
...
 
             <ul class="navbar-nav header__menu-list">
 
                 {% for menuitem in main_menu_items %}
+                <li class="nav-item">
+                    <a class="nav-link header__link{% if request.path == menuitem.url %} active{% endif %}" href="{{ menuitem.url }}">
+                        {{ menuitem.translated_menu_title }}
+                    </a>
                     {% if menuitem.subpages %}
+                    {% if menuitem.url in request.path %}
```
Comment (phillimorland) [inline apps/core/templates/navbar-simple.html:40]: tried combining these but it wasn't happy but might be a morning brain thing

**Review by goapunk — APPROVED** [review]
Looks good so far. Only issue is that if there are a lot of menu items they will exceed the screen on mobile at some point and become unreachable as you can't scroll the hamburger menu. With the current amount it works fine with the smallest screen size so maybe we ignore it for now?

**Comment by phillimorland** [comment]
@goapunk 
- have set max number to 5 which worked on my testing but can be tested by PM and design
- due to our original page structure dev does not represent correct parent child structure of urls, I had a bit of a play to limit the links one can choose to just children of the main menu item but this seems problematic https://github.com/wagtail/wagtail/issues/2756 so think we have to just trust the editors to implement correct structure, I will set up dev correctly so there is a blueprint for them.

**Review by goapunk — APPROVED** [review]
cool

**Pull request merged**